src/rosdiscover/cli.py

```diff
@@ -58,12 +59,29 @@ def generate_acme(args):
     if args.acme is None:
         print(acme)
         if args.check:
-            acme_gen.check_acme(acme)
+            acme_gen.check_acme()
     else:
         if args.check:
             acme_gen.check_acme()
 
 
+def _observe(args) -> SystemSummary:
+    config = Config.from_yaml_string(args.config)
+    with Observer.for_container(args.container, config) as obs:
```

Comment:
If we drop `contextmanager` from `for_container`, it should be fine to just do:
```
observer = Observer.for_container(args.container, config)
summary = observer.observe()
```

Reply:
Done